c2rust-transpile/src/translator/literals.rs

```diff
@@ -72,6 +72,18 @@ impl<'c> Translation<'c> {
         mk().cast_expr(value, target_ty)
     }
 
+    /// Return whether the literal can be directly translated as this type
+    pub fn literal_matches_ty(&self, kind: &CLiteral, ty: CQualTypeId) -> bool {
```

Comment:
Is there a reason a `CLiteral` is named `kind`?

Reply:
`lit` is probably a better name. Will change.